### PR TITLE
[opentelemetry/guide] Add guide for collecting host metadata via semantic conventions

### PR DESCRIPTION
**Pull request by mx-psi**
### What does this PR do? What is the motivation?

Adds guide for mapping OpenTelemetry semantic conventions into the infrastructure list

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by mx-psi** [comment]
This PR is reviewable, but please don't merge just yet, I want to sync with Datadog support before merging this :)

File: content/en/opentelemetry/guide/host_metadata.md
Comment (liustanley) [inline content/en/opentelemetry/guide/host_metadata.md:55]: nit: Disregard if you think this is unnecessary, but this line could be made into a heading for navigation and clearer division of the tables. Same for lines 62 and 76.
```suggestion
### General system conventions
```

Comment (mx-psi) [reply content/en/opentelemetry/guide/host_metadata.md]: Looks better now, thanks for the suggestion

File: content/en/opentelemetry/guide/host_metadata.md
Comment (mx-psi) [reply content/en/opentelemetry/guide/host_metadata.md]: Done!

**Review by liustanley — APPROVED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**
